components/resources/brave_news_strings.grdp

```diff
@@ -69,6 +69,18 @@
   <message name="IDS_BRAVE_NEWS_SUGGESTIONS_SUBTITLE" desc="Subtitle of the suggested sources section">
     Some sources you might like. Suggestions are anonymous and matched only on your device.
   </message>
+  <message name="IDS_BRAVE_NEWS_ERROR_HEADING" desc="Message heading for when there is a Brave News error">
+    Oops…
+  </message>
+  <message name="IDS_BRAVE_NEWS_ERROR_MESSAGE" desc="Message body for when there is a Brave News error">
+    Brave News is experiencing some issues. Try again.
+  </message>
+  <message name="IDS_BRAVE_NEWS_NO_CONTENT_HEADING" desc="Message heading for when there is no Brave News content">
+    No content
+  </message>
+  <message name="IDS_BRAVE_NEWS_NO_CONTENT_MESSAGE" desc="Message body for when there is no Brave News content">
+    Brave News has no content to show. Go follow some sources!
```

Comment:
Completely & utterly optional: I wonder if it's worth having a button show on the card to open the Brave News dialog?

Reply:
Added the button
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/741836/200757530-1a36c0d8-4db1-456a-82cd-49cac9505dcb.png">